modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/runtime/resolver/ParameterGroupArgumentResolver.java

```diff
@@ -36,7 +33,7 @@ public ParameterGroupArgumentResolver(ParameterGroupDescriptor group, Reflection
   public LazyValue<T> resolve(ExecutionContext executionContext) {
     return new LazyValue<>(() -> {
       try {
-        return new ParameterGroupObjectBuilder<T>(group, reflectionCache).build((EventedExecutionContext) executionContext);
+        return builder.build((EventedExecutionContext) executionContext);
```

Comment:
the builder is not stateless, so reusing it may break something

Reply:
fixed